### ISSUE TITLE
[completed-docs] group exports are not checked

### ISSUE THREAD

**Issue by mctep** [palantir/tslint#4510]
### Bug Report

- __TSLint version__: 5.12.0
- __TypeScript version__: 3.2.2
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
export const b = 1;
const a = 1;
export { a };
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "completed-docs": [
      true,
      {
        "variables": { "visibilities": ["exported"] }
      }
    ]
  }
}
```

#### Actual behavior
```
ERROR: ./test.ts:1:14 - Documentation must exist for exported variables.
```
#### Expected behavior
```
ERROR: ./test.ts:1:14 - Documentation must exist for exported variables.
ERROR: ./test.ts:3:10 - Documentation must exist for exported variables.
```

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
